homeassistant/components/octoprint/manifest.json

```diff
@@ -3,7 +3,7 @@
   "name": "OctoPrint",
   "config_flow": true,
   "documentation": "https://www.home-assistant.io/integrations/octoprint",
-  "requirements": ["pyoctoprintapi==0.1.6"],
+  "requirements": ["pyoctoprintapi==0.1.7"],
```

Comment:
I think they're going to want a separate PR for the dependency upgrade

Reply:
Wasn't sure if I could bundle it with this change as the bump only effects the endpoints this change uses (and one unused endpoint for a future PR).